cpp/src/table/table_rowwise_operators.cuh

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright 2019 BlazingDB, Inc.
+ *     Copyright 2019 William Scott Malpica <william@blazingdb.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
Seems this file should be called "device_table_row_operators.cuh" since the operators are for device_table, not table.